charts/shoot-core/charts/csi-packet/templates/controller.yaml

```diff
@@ -0,0 +1,88 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-packet-pd
+  labels:
+    app: csi-packet-pd
+spec:
+  selector:
+    app: csi-packet-pd
+  ports:
+    - name: dummy
+      port: 12345
+
+---
+
+kind: StatefulSet
+apiVersion: apps/v1beta1
+metadata:
+  name: csi-packet-controller
+  namespace: kube-system
+spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      partition: 0
+  serviceName: "csi-packet-pd"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-packet-pd-driver
+  template:
+    metadata:
+      labels:
+        app: csi-packet-pd-driver
+    spec:
+      serviceAccount: csi-controller-sa
+      containers:
+        - name: csi-external-provisioner
+          imagePullPolicy: IfNotPresent
+          image: quay.io/k8scsi/csi-provisioner:v1.0.1
```

Comment:
This should be injected from the image vector and not hard-coded here, see e.g. https://github.com/gardener/gardener/blob/master/charts/shoot-core/charts/csi-alicloud/templates/csi-attacher-dep.yaml#L31

Reply:
Good! I didn't like it hard-coded. Thank you.